src/models/document-content.ts

```diff
@@ -1,9 +1,9 @@
 import { types } from "mobx-state-tree";
 
-export const AuthoredContentModel = types
+export const DocumentContentModel = types
   .model("AuthoredContent", {
```

Comment:
If the name does change, the model name needs an update.

Reply:
Doh! Then you should have marked it as requiring changes so that I could've fixed it before you merged it.